readthedocs/projects/views/private.py

```diff
@@ -480,7 +487,7 @@ def get_form(self, data=None, files=None, **kwargs):
         return super().get_form(data, files, **kwargs)
 
 
-class ProjectUsersList(SuccessMessageMixin, ProjectUsersMixin, FormView):
+class ProjectUsersList(ProjectUsersMixin, FormView):
```

Comment:
Does `ProjectUsersMixin` also handles the success message? Or in this case it comes from `FormView`?

Reply:
Yea, ProjectUsersMixin  inherits from ProjectAdminMixin.